docker-plugin/src/main/java/io/micronaut/gradle/docker/NativeImageDockerfile.java

```diff
@@ -69,6 +72,12 @@ public abstract class NativeImageDockerfile extends Dockerfile implements Docker
     @Input
     public abstract Property<String> getGraalVersion();
 
+    /**
+     * @return The Graal architecture
+     */
+    @Input
+    public abstract Property<String> getGraalArch();
```

Comment:
should this be `getArch` instead of `getGraalArch`?

Reply:
I think that `graalArch` makes sense indeed.